src/libs/utils.ts

```diff
@@ -230,9 +230,9 @@ export const computeWorkspaceError = ({ canCompute, workspace: { isLocked } }) =
   )
 }
 
-export const textMatch = _.curry((needle, haystack) => {
+export const textMatch = (needle: string, haystack: string): boolean => {
```

Comment:
Removing curry here breaks withAutocomplete

https://github.com/DataBiosphere/terra-ui/blob/9700a2df29dd48dcd0820acf9cad4679820f4170/src/components/input.js#L251-L255

Reply:
Hmm.. this is true. I changed it to safe curry for this function, but had to use some odd syntax to make to call site happy.

Reply:
@msilva-broad can you take a look at this. I wouldn't expect safeCurry to require a cast.